README.md

```diff
@@ -1,34 +1,55 @@
-<p align="center">
-  <img src="public/assets/img/logo-without-bg.png" width="320"/>
-</p>
-
+![OpenTech](https://i.ibb.co/4g2hgty/landing.png)
```

Comment:
This logo is different from the original, please return the old one. This logo looks weird:
![Screenshot from 2020-09-02 18-10-41](https://user-images.githubusercontent.com/28131369/92046028-a7afd200-ed47-11ea-97da-909996820890.png)

Reply:
Para que el badge no pierda el formato con el:
```
<p>
</p>
```
Se optó por la opción presentada, pero entiendo lo del logo. Se corregirá inmediatamente